packages/asset-swapper/src/utils/quote_requestor.ts

```diff
@@ -95,6 +98,7 @@ export class QuoteRequestor {
         marketOperation: MarketOperation,
         options?: Partial<RfqtRequestOpts>,
     ): Promise<SignedOrder[]> {
+        this._warningLogger('inside firm quotes');
```

Comment:
same below..

Reply:
removed in https://github.com/0xProject/0x-monorepo/pull/2562/commits/141c30b173881f1f09307d951d6d8a63483f732e